changelog/unreleased/bugfix-spaces-breadcrumbs

```diff
@@ -0,0 +1,8 @@
+Bugfix: Spaces breadcrumbs not working correctly
+
+We've fixed a bug, where clicking on the last breadcrumb navigation item not resulted
```

Comment:
No comma before `where` necessary in the English language 🤓 but also not too much of a problem